Gemfile

```diff
@@ -86,3 +86,6 @@ end
 
 # Windows does not include zoneinfo files, so bundle the tzinfo-data gem
 gem 'tzinfo-data', platforms: [:mingw, :mswin, :x64_mingw, :jruby]
+
+# twitter API
+gem "twitter", "~> 6.2"
```

Comment:
Let's move this commit before `chore: add twitter authentication `

Reply:
Oh whoops I think my commits were out of order :/